### PR TITLE
Aligning with changes in Structure_Engine

### PR DESCRIPTION
**Pull request by IsakNaslundBh**
### NOTE: Depends on 

https://github.com/BHoM/BHoM_Engine/pull/2146/
   
### Issues addressed by this PR

https://github.com/BHoM/BHoM_Engine/issues/2135

Aligning with methods being removed from Structure_Engine.

This is only adding a reference and changing a to target another pre-existing method, so can safely be merged before others.

### CODE REVIEW

File: MidasCivil_Engine/MidasCivil_Engine.csproj [hunk 1]
```diff
@@ -73,6 +73,11 @@
       <HintPath>C:\ProgramData\BHoM\Assemblies\Reflection_oM.dll</HintPath>
       <Private>False</Private>
     </Reference>
+    <Reference Include="Spatial_Engine">
+      <SpecificVersion>False</SpecificVersion>
```
Comment (peterjamesnugent) [inline MidasCivil_Engine/MidasCivil_Engine.csproj:77]: Remove specific version

**Review by peterjamesnugent — CHANGES_REQUESTED** [review]
Remove specific version in `.csproj` file.

**Comment by IsakNaslundBh** [comment]
@BHoMBot check project compliance

**Review by peterjamesnugent — APPROVED** [review]
LGTM.

**Pull request merged**